test/relativistic/relativistic_mass_test.py

```diff
@@ -0,0 +1,49 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    convert_to,
+)
+from symplyphysics.laws.relativistic import relativistic_mass
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    m = Quantity(6 * units.kilogram)
+    v = Quantity(2_000_000 * (units.meter / units.second))
+    Args = namedtuple("Args", ["m", "v"])
+    return Args(m=m, v=v)
+
+
+def test_basic_mass(test_args):
+    result = relativistic_mass.calculate_relativistic_mass(
+        test_args.m, test_args.v)
+    result_mass = convert_to(result, units.kilogram).evalf(4)
+    assert result_mass == approx(6.04, 0.01)
```

Comment:
Where did you get this value. It is preferable to use some value from a book or exercise, to verify the correctness of the law.

Reply:
I calculated using calculator